policy.go

```diff
@@ -67,6 +67,16 @@ type policyOrDataNode struct {
 
 type policyOrDataTree []policyOrDataNode
 
+type dynamicPolicyDataRaw_v0 dynamicPolicyData
+
+func (d *dynamicPolicyDataRaw_v0) data() *dynamicPolicyData {
+	return (*dynamicPolicyData)(d)
+}
+
+func makeDynamicPolicyDataRaw_v0(data *dynamicPolicyData) *dynamicPolicyDataRaw_v0 {
```

Comment:
this name puzzled me a bit, but I suppose in general this wouldn't be a cast but produce a different *struct, right? so "make" makes sense

Reply:
Yeah, I did it this way because once the metadata format is changed, this will do something other than cast.